tox.ini

```diff
@@ -33,73 +30,24 @@ deps =
     pytest-rerunfailures==9.0
 
 commands =
-    pip install git+https://github.com/pytoolz/cytoolz.git#egg=cytoolz==0.10.1.dev0
+    ; ignore exit code of tensorflow because it does not work on Python 3.8
```

Comment:
It might be worth checking if we can upgrade the ml example to 2.+ of tensorflow without any work